learning_journal/static/styles.css

```diff
@@ -66,8 +73,35 @@ main aside .field textarea{
 main aside .control_row input{
     margin-left:16%}
 footer{
-    padding: 1em 0.75em;
-    background: #222;
-    color: #ccc;
-    border-top: 3px solid #fff;
+    position: absolute;
+    bottom: 0;
+    padding: .2em 0.75em;
+    background: #fff;
+    color: black;
+    width: 100%;
+    border-top: 1px solid black;
     border-bottom: 3px solid #fff;}
+strong {
+    color: red;
+    text-decoration: underline;
+    font: 16px "Courier New", Courier, monospace;}
+p {
+    font: 16px "Courier New", Courier, monospace;}
+input[type=submit] {
+    font: 14px "Courier New", Courier, monospace;}
+input{
+    font: 16px "Courier New", Courier, monospace;}
+textarea{
+    font: 16px "Courier New", Courier, monospace;}
```

Comment:
FYI, it's common to put shared rules like this font rule into a single declaration, and then add a few extra ones to change only those parts that are different from tag to tag:

```css
strong, p, input, textarea, nav {
    font: 16px "Courier New", Courier, monospace;
}
strong {
    color: red;
    text-decoration: underline;
}
input[type=submit] {
    font-size: 14px;
}
...
```